Earthfile

```diff
@@ -88,7 +88,7 @@ docker:
     ARG TARGETARCH
     ARG TARGETVARIANT
 
-    FROM --platform=$TARGETPLATFORM ubuntu:20.04
+    FROM --platform=$TARGETPLATFORM ubuntu:25.10
```

Comment:
this should be LTS and `25.10` is not. you could use `ubuntu:latest` since that uses the latest LTS image or `ubuntu:24.04`